contrib/partitioner/partitioner.py

```diff
@@ -94,17 +94,21 @@ def main():
     else:
         print('Data mount exists')
 
-    # Check for more than 1 partition
+    # Check for more than 1 partition (and one drive only)
     # if so get rid of it all
-    if len(usb_partitions()) > 1:
-        print('More than one partition found! Lets fix this up');
-        partitions = usb_partitions();
-        partitions_new = [];
-        for partition in partitions:
-            partitions_new.append(partition.replace('sda',''))
-        
-        for pn in partitions_new:
-            os.system('parted -s /dev/sda rm ' + pn);
+    if len(usb_devs()) == 1:
+        if len(usb_partitions()) > 1:
+            print('More than one partition found! Lets fix this up');
+            partitions = usb_partitions();
+            partitions_new = [];
+            for partition in partitions:
+                partitions_new.append(partition.replace('sda',''))
+
+            for pn in partitions_new:
+                os.system('parted -s /dev/sda rm ' + pn);
+    else:
+        print('More than one drive is not supported');
+        sys.exit(1);
```

Comment:
Also just a tiny nit but might be neater to flip this condition so then there's no need for else.

like:
```python
if len(usb_devs()) != 1:
    print('Only one drive is supported');
    sys.exit(1);

# No need to indent rest of code inside else
```

Helps keep the code more readable and easier to follow. But this is a subjective change so feel free to disagree.

Reply:
Sure, happy to flip it

Eventually I wanted to get this out of contrib/ anyway and into umbrel-os rather than this project. Once we get this one working for 99% of all setups, we can move this into umbrel-os and tidy things more.